src/net.cpp

```diff
@@ -2092,6 +2093,38 @@ bool CConnman::BindListenPort(const CService &addrBind, std::string& strError, b
     return true;
 }
 
+bool CConnman::BindUNIX(const boost::filesystem::path &path, std::string& strError, bool fWhitelisted)
+{
+#ifdef HAVE_SOCKADDR_UN
+    // Bind on UNIX socket
+    int fd = evunix_bind_fd(path);
+    if (fd < 0)
+    {
+        strError = "Unable to bind to UNIX socket " + path.string();
+        LogPrintf("%s\n", strError);
+        return false;
+    }
+    SOCKET hListenSocket = (SOCKET)fd;
+    LogPrintf("P2P bound to %s\n", path.string());
+
+    // Listen for incoming connections
+    if (listen(hListenSocket, SOMAXCONN) == SOCKET_ERROR)
+    {
+        strError = strprintf(_("Error: Listening for incoming connections failed (listen returned error %s)"), NetworkErrorString(WSAGetLastError()));
```

Comment:
In commit "p2p: Bare minimum to support UNIX sockets"

Any reason this error is translated while others are not?